### DISCUSSION TITLE
NPM Run Build Error 126

### DISCUSSION THREAD

**Discussion by SoulTech9** [vercel/community#4359] [Help]
### Summary

![image](https://github.com/vercel/community/assets/112087610/3c651f29-054f-4d1a-b0a1-008874af37c5)
![image](https://github.com/vercel/community/assets/112087610/18084f2a-56c5-405f-b0d7-57444a17069f)
![image](https://github.com/vercel/community/assets/112087610/8587d6d1-8d75-43c5-8a04-b0d2b6a41306)
![image](https://github.com/vercel/community/assets/112087610/dd4f732f-d970-4ad7-8359-4694932f477b)
![image](https://github.com/vercel/community/assets/112087610/fda2092c-8357-4ec1-b21a-7ac14fed6a0e)

Hi I am getting this error and I don't know why, can anybody spot what is wrong with my React + Vite app?

### Steps to Reproduce

Everytime I try to deploy it gets an error... It works locally

**Comment by amyegan** [comment]
Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that we can use to help with debugging?
